workflows/diagnostics/README.md

```diff
@@ -4,7 +4,7 @@ Compute summary diagnostics and metrics from the output of a prognostic run and
 visualize on report comparing multiple runs.
 
 See the [Sphinx
-Documentation](https://vulcanclimatemodeling.com/docs/prognostic_run_diags/)
+Documentation](https://vulcanclimatemodeling.com/docs/diagnostics/)
```

Comment:
update this README to reflect that this package now does offline diagnostics too. Maybe could just copy in the offline README to the end of this one, and leave the Sphinx docs to just describe the online diags?